waves/project.py

```diff
@@ -1472,31 +1508,33 @@ def energy_production(
         if frequency not in opts:
             raise ValueError(f"`frequency` must be one of {opts}.")  # type: ignore
 
-        # For the wind rose outputs, only consider project-level availability because
-        # wind rose AEP is a long-term estimation of energy production
+        # Use WOMBAT availability for the base index
         availability = self.wombat.metrics.production_based_availability(
             frequency="month-year", by="turbine"
         ).loc[:, self.floris_turbine_order]
+
         if self.floris_results_type == "wind_rose":
-            power = pd.DataFrame(0.0, dtype=float, index=availability.index, columns=["drop"])
-            power = power.merge(
+            energy_gwh = pd.DataFrame(0.0, dtype=float, index=availability.index, columns=["drop"])
```

Comment:
@RHammond2 fix this to get potential as the base energy, so losses can be subtracted.